fhir-validation/src/main/java/com/ibm/fhir/validation/FHIRValidator.java

```diff
@@ -168,6 +170,7 @@ private FHIRValidator() { }
             throw new IllegalArgumentException("Root must be resource node");
         }
         try {
+            evaluationContext.setExternalConstant(ResolveFunction.RESOLVE_RELATIVE_REFERENCES, FHIRPathBooleanValue.TRUE);
```

Comment:
are you sure you want to set this to true?

Reply:
could it be an unexpected side-effect of callers that have passed an evaluation context?

Reply:
Yes.

Reply:
> are you sure you want to set this to true?
john explained to me that we have 2 layers to this opt-in.  setting RESOLVE_RELATIVE_REFERENCES to true is just the first layer.  it still won't actually resolve those references unless the corresponding fhir-server-config property is set such that we use the ServerResolveFunction and not the default ResolveFunction.

Reply:
as for the side-effect concern, john informs me that we already use this pattern in a few other spots.
setting an external constant more than once shouldn't cause any issues, so there's no concern for re-using the same context for additional validations.